frontend/fingertips-frontend/components/views/OneIndicatorOneAreaView/index.tsx

```diff
@@ -43,6 +44,11 @@ export default async function OneIndicatorOneAreaView({
   await connection();
   const indicatorApi = ApiClientFactory.getIndicatorsApiClient();
 
+  const areaTypeToUse =
+    areaCodes.length === 1 && areaCodes[0] === areaCodeForEngland
+      ? englandAreaType.key
+      : areaTypeSelected;
```

Comment:
I think, given the block above that will throw an error if `areaCodes.length !==1`, we can assume we only have a single area code in this view.
```suggestion
  const areaTypeToUse =
    areaCodes[0] === areaCodeForEngland
      ? englandAreaType.key
      : areaTypeSelected;
```